utils/build-windows.bat

```diff
@@ -307,16 +310,21 @@ cmake "%source_root%\swift-corelibs-libdispatch"^
     -DCMAKE_BUILD_TYPE=%CMAKE_BUILD_TYPE%^
     -DCMAKE_C_COMPILER=clang-cl^
     -DCMAKE_CXX_COMPILER=clang-cl^
-    -DCMAKE_SWIFT_COMPILER:PATH=%install_directory%\bin\swiftc.exe^
+    -DCMAKE_SWIFT_COMPILER=swiftc^

```

Comment:
libdispatch is using CMake 3.15, the argument is `CMAKE_Swift_COMPILER`.  Yes, the case matters ... CMake.

Reply:
Seems that having it in the path is enough? Seems to be working :D In any case I will change it.